pkg/detectors/bingsubscriptionkey/bingsubscriptionkey.go

```diff
@@ -0,0 +1,95 @@
+package bingsubscriptionkey
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+
+	regexp "github.com/wasilibs/go-re2"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct {
+	client *http.Client
+}
+
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	defaultClient = common.SaneHttpClient()
+	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"bing"}) + `\b([a-fA-F0-9]{32})\b`)
+)
+
+func (s Scanner) Keywords() []string {
+	return []string{"bing", "subscription"}
```

Comment:
IMO `subscription` as keyword is very common. Either we can remove it or update it to something like `bing_subscription`

Reply:
removed